app/resources/generated_resources_en-GB.xtb

```diff
@@ -1976,7 +1976,7 @@ Keep your key file in a safe place. You will need it to create new versions of y
 <translation id="5260334392110301220">Smart Quotes</translation>
 <translation id="7121728544325372695">Smart Dashes</translation>
 <translation id="2653275834716714682">Text Replacement</translation>
-<translation id="2845382757467349449">Always Show Bookmarks Bar</translation>
```

Comment:
@mkarolin Not sure why the ID has changed here if you recently did a pull from transifex to this branch. Any ideas?